nilearn/datasets/func.py

```diff
@@ -506,6 +506,9 @@ def fetch_miyawaki2008(data_dir=None, url=None, resume=True, verbose=1):
         - 'mask': string
             Path to nifti mask file to define target volume in visual
             cortex
+        - 'bg': string
```

Comment:
> It would be consistent but since it's not really an anatomical image acquired
> with the dataset, I'm not sure that we can do that.

OK. I am fine with 'background'.